src/location.ml

```diff
@@ -8,7 +8,7 @@ type t = location = {
 }
 
 let in_file name =
-  let loc = { pos_fname = name; pos_lnum = 1; pos_bol = 0; pos_cnum = -1 } in
+  let loc = { L.dummy_pos with pos_fname = name } in
```

Comment:
I think this is equivalent to the compiler's `in_file`. If we used it instead, we wouldn't need the cinaps bits for dummy_pos anymore.